src/blenderbim/blenderbim/core/qto.py

```diff
@@ -21,3 +21,22 @@ def calculate_circle_radius(qto, obj=None):
     result = qto.get_radius_of_selected_vertices(obj)
     qto.set_qto_result(result)
     return result
+
+def assign_pset_qto(qto, selected_objects):
+    for object in selected_objects:
+        qto.set_active_object(object)
+        qto.assign_pset_qto_to_selected_object(object)
```

Comment:
I don't understand this core function. All it does is assign a qto. Why isn't this "built-in" into the existing add_qto function?

Reply:
Is it better if the name of this function was "_assign_predefined_pset_qto_"?
Because this function also gets the predefined pset_qto based on the ifc class assigned to the object..
Am i missing something? :-)